Sources/StreamChat/APIClient/APIClient.swift

```diff
@@ -167,19 +158,10 @@ class APIClient {
     /// - Parameters:
     ///   - endpoint: The `Endpoint` used to create the network request.
     ///   - completion: Called when the networking request is finished.
-    private func executeRequest<Response: Decodable>(
+    func executeRequest<Response: Decodable>(
```

Comment:
When a user is connected as a `guest` token refresh is done using stream backend. The thing is, when token expired error is reported during **connection recovery flow** when request queue in APIClient is suspended we make this request off the queue so it can pass through.